### DISCUSSION TITLE
How to ask or grant permission to use the webcam?

### DISCUSSION THREAD

**Discussion by jafrancov** [r0x0r/pywebview#723] [Q&A]
Hi, what's the best practice to get the permission to use the webcam?, I have tried to get the devices with navigator.mediaDevices.enumerateDevices, however, it crashes because the lack of permissions, if anyone could point me in the right direction, I would appreciate it!

**Comment by r0x0r** [comment]
what platform/renderer are you using? my guess is that web cam permissions must be explicitly set in the renderer settings.
